src/Notepads/Controls/FindAndReplace/FindAndReplaceControl.xaml

```diff
@@ -4,81 +4,156 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:querybox="using:Notepads.Controls.FindAndReplace"
     mc:Ignorable="d"
     d:DesignHeight="72"
     d:DesignWidth="250">
 
-    <Grid x:Name="RootGrid">
+    <Grid x:Name="RootGrid" KeyDown="RootGrid_KeyDown">
```

Comment:
Name it "FindAndReplaceRootGrid" and change event name to "FindAndReplaceRootGrid_OnKeyDown"